man/SAPHanaSR_maintenance_examples.7

```diff
@@ -649,7 +674,7 @@ F.Herschel, L.Pinne.
 .SH COPYRIGHT
 (c) 2017-2018 SUSE Linux GmbH, Germany.
 .br
-(c) 2019-2024 SUSE LLC
+(c) 2019-2023 SUSE LLC
```

Comment:
I think it's unintentional to go back in time. So can you please correct the date.